ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -45,34 +45,31 @@
     </h2>
     <h2 class="title is-3 grid-item-right-bottom" data-test-connection>{{connection}}</h2>
   {{else}}
-    <h3 class="title is-5 grid-item-title card-title">{{title}}
-      <ToolTip @verticalPosition="above" @closeDelay="2000ms" as |T|>
-        <T.trigger>
-          <Icon @glyph={{"check-circle-outline" }} @size="m" class={{"has-text-grey"}} />
-        </T.trigger>
-        <T.content @class="tool-tip">
-          <div class="box">
-            There may be a difference between last remote WAL and last WAL entry from primary. If it persists over time,
-            investigate.
-            <LearnLink @path="/vault/operations/monitor-replication#are-my-dr-clusters-in-sync" @class="has-text-white">
-              Learn more
-            </LearnLink>
-          </div>
-        </T.content>
-      </ToolTip>
-    </h3>
+    <h3 class="title is-4 grid-item-title card-title">{{title}}</h3>
     <div class="grid-item-left">
-      <h6 class="title is-6">Delta</h6>
-      <p class="has-text-grey is-size-8">The difference between the last remote WAL and the last WAL entry from the primary.</p>
+      <h6 class="title is-6">last_remote_wal</h6>
+      <p class="has-text-grey is-size-8">The last WAL index that the secondary received from the primary. Updates every 10 seconds.</p>
     </div>
     <div class="grid-item-right">
-      <h6 class="title is-6">last_wal</h6>
-      <p class="has-text-grey is-size-8" data-test-lastWAL>{{lastWAL}}</p>
+      <h6 class="title is-6">primary_cluster_addr</h6>
+      {{#if primaryClusterAddr}}
+        <span>
+          <a
+            href={{primaryClusterAddr}}
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener" 
+          >
+            {{primaryClusterAddr}}
```

Comment:
this is interesting -- this link doesn't actually work for me because it's taking me to `https://127.0.0.1:8201` (presumably the cluster address itself) instead of `http://localhost:4200/ui/` (the UI server). if the intention of this link is to be able to click through to the other UI, how do we use the cluster address to find the UI? we are going to need to solve this problem for the known secondaries table as well. 🤔

Reply:
related: this scenario should be included in the acceptance tests somehow...

Reply:
Good call.  Let me see if I can dig up any info on this and I'll also try and add an acceptance test for it.